example.yaml

```diff
@@ -151,6 +203,38 @@ aws_secret_access_key: ""
 aws_region: ""
 aws_s3_bucket: ""
 
-# == LocalFile Output ==
+# == Local file Output ==
```

Comment:
So, this was written as `LocalFile` because that's how we've named the plugin: https://github.com/stripe/veneur/tree/master/plugins/localfile